demisto_sdk/commands/test_content/tests/DemistoClientMock_test.py

```diff
@@ -0,0 +1,113 @@
+import re
```

Comment:
dont think this file should have the `_test.py` suffix in its name

Reply:
The precommit screamed at me for it.
Do you want me to try figuring out how to ignore it? or leave it as is?